reader/components/header/Header.jsx

```diff
@@ -23,13 +23,15 @@ const Header = () => {
     <div className="header">
       <div className="item d-flex justify-content-start">
         <Button
+          title="Show Document Outline"
```

Comment:
Try to make it simpler. _Show outline_ might be better. What do you think about?

Please, follow the same case. Captions in the bottom are in sentence case but this one is in title case. I would prefer sentence.